dracut/30ignition/ignition-setup-user.service

```diff
@@ -10,12 +10,10 @@ Before=ignition-disks.service
 Before=ignition-files.service
 
 # On diskful boots, ignition-generator adds Requires/After on
-# dev-disk-by\x2dlabel-boot.device.
+# Requires=mnt-boot.mount
+# After=mnt-boot.mount
```

Comment:
Hmm, is that still true though? I don't see the generator add this anymore.

I think we just want `After=mnt-boot.mount` here, right? That way *if* we're in a diskful boot, then we're guaranteed to run after `/mnt/boot` is mounted.

Reply:
Ouuhh actually, it might be even better here to use `RequiresMountsFor=/mnt/boot`. That way we don't even bother running at all if the mount unit failed.

Reply:
Yeah, that's cruft.